### ISSUE TITLE
ugly indentation in lilypond output

### ISSUE THREAD

**Issue by mtarenskeen** [Philomelos/lilypond-musicxml2ly-dev#2]
The indentation and formatting that musicxml2ly outputs is really wrong and ugly. Frescobaldi automatically reformats this (using python-ly) but the musicxml2ly output should be goodlooking also for people who don't use frescobaldi.

**Comment by Philomelos** [comment]
Yes, in theory you are right.  In practice there are loads of other bugs that might prevent LilyPond from compiling the converted files successfully.  So the beauty of output still isn't the topmost priority.  It probably shouldn't be too hard to let musicxml2ly-dev call python-ly automatically, though.  Would that help?

**Comment by mtarenskeen** [comment]
It's not a big deal. Frescobaldi uses musicxml2ly + python-ly when importing MusicXML files. And if I am using something else, for example vim, I can call python-ly myself if I want to. Better not draw in more dependencies for such a simple tool. (simple usage, not simple development :-).
I hope your bugfixes and improvements will be merged in the official LilyPond development tree soon. Thanks for your efforts.
